deployment/config.js

```diff
@@ -1,7 +1,6 @@
-const {EnvKeys, EnvObject} = require('./env');
+const {EnvKeys, EnvObject} = require('./config-env');
```

Comment:
Note that the rename is technically a breaking change, cause of downstream usage i.e. https://github.com/zeit/now-create/blob/2948cc5e2e1ea0139abbc37f024b6ff3c4de6f99/lib/validate-request-body.js#L3

Reply:
👍 will update